cloudformation/media-atom-maker.json

```diff
@@ -55,6 +55,10 @@
             "Description": "Name of the upload tracking dynamo table",
             "Type": "String"
         },
+        "ManualPlutoTable": {
```

Comment:
I'm not sure if this is something we should be doing instead of just adding to the s3 secrets file. @akash1810 do you know a good reason for doing this?